src/test/util/setup_common.h

```diff
@@ -171,7 +188,7 @@ struct TestChain100Setup : public TestingSetup {
      * @param submit             Whether or not to submit to mempool
      */
     CMutableTransaction CreateValidMempoolTransaction(CTransactionRef input_transaction,
-                                                      int input_vout,
+                                                      uint32_t input_vout,
                                                       int input_height,
```

Comment:
cea7dcf4388ad65436fcb98c7025b607708e09ab
nit: worth changing `input_height` to `uint` too, maybe even with `const`?

Reply:
- It will need to be cast to `int` anyway when we call `AddCoins()`. Maybe one day if we change `AddCoins()` ?
- I don't think `const` is very meaningful since it's passed by value